scripts/postgres/restoreDbFromSource.ts

```diff
@@ -75,8 +76,11 @@ async function main() {
     username: sourceUsername,
   });
 
+  const sanitizedFileName = `sanitized-${backUpFileName}`;
+  replaceEmailAddresses(backUpFileName, sanitizedFileName);
```

Comment:
I'm not sure this really matters, but I suspect there is a way to do all the redaction in memory rather than storing the unsanitized data at all, something like removing the `-f` flag in `pg_dump` (so that `pg_dump` uses STDOUT) and then piping through the redactor function. (I realize this is a draft, I was just curious 😄 .)